code/modules/hydroponics/grown.dm

```diff
@@ -60,6 +60,31 @@
 
 /obj/item/reagent_containers/food/snacks/grown/attackby(obj/item/O, mob/user, params)
 	..()
+	if(slices_num && slices_num >0 && slice_path)
```

Comment:
if `slices_num` is above 0 then it would be true so `&& slices_num > 0` is redundant